src/main/resources/db/migration/V5_1__add_column_is_sendt_to_arena.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE DIALOGMELDINGOPPLYSNINGER
+    ADD COLUMN is_sent_to_arena BOOLEAN DEFAULT FALSE;
```

Comment:
🍦 til alle i Arena!!

Reply:
Tror kanskje ikke vi pleier å prefikse boolske databasekolonner med :ice_cream: Har ikke så sterke meninger om det selv... kan hende det er lurt, men det er jo også gitt av typen og man kan få litt lange navn :man_shrugging:

Reply:
Nei jeg er helt åpen for å endre navn hvis andre har bedre forslag 😄